src/components/sitepages/SiteTargets.vue

```diff
@@ -48,7 +48,9 @@ import $ from 'jquery'
 import TheSkyChart from '@/components/celestialmap/TheSkyChart'
```

Comment:
I think my previous comment about potentially including skychart legend belongs here instead. A symbol legend could be useful probably not on the skychart itself but somewhere on this page if there is reasonable space.